includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -66,10 +65,11 @@ public function __construct( Context $context ) {
 	 * Registers functionality through WordPress hooks.
 	 *
 	 * @since 1.18.0.
+	 * @since n.e.x.t Renamed hooks to target Analytics 4 module.
 	 */
 	public function register() {
 		add_action(
-			'googlesitekit_analytics_init_tag',
+			'googlesitekit_analytics_4_init_tag',
```

Comment:
These hooks are incorrect since the `analytics_4` is interpolated from the module slug which is not the same:
https://github.com/google/site-kit-wp/blob/bd5892c89112179b54a8f9eda69f18564bac767f/includes/Core/Modules/Tags/Module_Web_Tag.php#L102

Reply:
If I'm understanding the issue right, we're best off using the actual module slug in here, so I've done that in this PR: #8074

Reply:
:facepalm: good spot, thanks @aaemnnosttv!